crates/turbopack-trace-utils/src/trace_writer.rs

```diff
@@ -1,10 +1,11 @@
 use std::{debug_assert, io::Write, thread::JoinHandle};
 
-use crossbeam_channel::{unbounded, Sender, TryRecvError};
+use crossbeam_channel::{bounded, unbounded, Receiver, Sender, TryRecvError};
 
 #[derive(Clone, Debug)]
 pub struct TraceWriter {
     channel: Sender<Vec<u8>>,
+    back_channel: Receiver<Vec<u8>>,
```

Comment:
```suggestion
    data_tx: Sender<Vec<u8>>,
    return_rx: Receiver<Vec<u8>>,
```

the field name should probably include whether it's a sender or receiver